docs/capabilities.md

```diff
@@ -0,0 +1,299 @@
+# Capabilities
```

Comment:
i have genuinely no idea where these came from as they don't appear to have previously been on either public or private repo. but they're great docs

Reply:
These were intentionally deleted from this repo to prefer the documentation on our public docs site so we should not reintroduce them here.